torch/csrc/distributed/rpc/functions.cpp

```diff
@@ -50,6 +59,47 @@ Message processRequestBlocking(Message&& request) {
       }
       break;
     }
+    case MessageType::REMOTE_CALL: {
+      ScriptRemoteCall src = ScriptRemoteCall::fromMessage(request);
+
+      auto rrefId = RRefId::fromIValue(src.retRRefId());
+      auto forkId = ForkId::fromIValue(src.retForkId());
+      TORCH_CHECK(rrefId != forkId, "Does not support remote call to self.");
+
+      auto& ctx = RRefContext::getInstance();
+      auto ownerRRef = ctx->getOrCreateOwnerRRef<IValue>(rrefId);
```

Comment:
Why is this created here but not passed the operation? I would assume that the rref that captures the result of the operation is something that _could_ be forwarded. If it isn't (yet, or ever), then the owner rref creation can be delayed until after the operation has completed successfully.

Reply:
In this PR, it can be moved to after the operation has completed. In the next one, it will do the following:

1. create an owner RRef
2. record the user RRef (the RRef on the caller)
3. send an `RREF_USER_ACCEPT` message back to the caller, so that the caller may delete it's local RRef after that. 
4. run the operation.

Hence, it needs the owner RRef before the operation execution. 

Let me move it to when the operation is done in this PR.